backend/graphql/resolvers.ts

```diff
@@ -0,0 +1,166 @@
+import { getGenres, getTracks, createTrack, updateTrack, getTrackById, deleteTrack, deleteMultipleTracks, saveAudioFile, deleteAudioFile } from '../utils/db';
```

Comment:
Would be nice to have linter built in to manage line breaking so there is no so long lines.